wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/subordinate/SubordinateBuilderComplexExample.java

```diff
@@ -139,7 +139,6 @@ private CreditCardPanel() {
 		 */
 		private WImage createCardImage(final String cardType) {
 			WImage image = new WImage();
-			image.setAccessibleText(cardType);

```

Comment:
Please replace setAccessibleText with setAlternativeText rather than deleting it.